app/src/main/java/com/yeonae/chamelezone/view/course/CourseModifyActivity.kt

```diff
@@ -38,21 +35,34 @@ class CourseModifyActivity : AppCompatActivity(), CourseModifyContract.View,
     private val placeNumbers = mutableListOf<Int>()
     private var isCreated = false
     private var imageNumber: Int = 0
+    private var savedImageName: String = ""
 
     override fun showMessage(response: Boolean) {
-        if(response){
+        if (response) {
             this.shortToast(R.string.success_update_course)
             finish()
         }
     }
 
     override fun showCourseDetail(courseList: List<CourseResponse>) {
+        savedImageName = courseList[0].courseImage
         firstPlaceNumber = courseList[0].placeNumber
         secondPlaceNumber = courseList[1].placeNumber
         imageNumber = courseList[0].courseImageNumber
         edt_course_title.text = SpannableStringBuilder(courseList[0].title)
         edt_course_content.text = SpannableStringBuilder(courseList[0].content)
 
+        imageContainer.removeAllViews()
+        val rlSlideImg = LayoutInflater.from(this).inflate(
```

Comment:
```suggestion
        val ivSlideImg = LayoutInflater.from(this).inflate(
```
type이 ImageView 이니 변수명이 rl 이면 안될 것 같아요.

Reply:
수정했습니당
f7dcf35